schema-loader/src/main/java/com/scalar/db/schemaloader/ImportTableSchema.java

```diff
@@ -0,0 +1,40 @@
+package com.scalar.db.schemaloader;
+
+import com.google.gson.JsonObject;
+import javax.annotation.concurrent.Immutable;
+
+@Immutable
+public class ImportTableSchema {
+  private static final String TRANSACTION = "transaction";
+  private final String namespace;
+  private final String tableName;
+  private final boolean isTransactionTable;
+
+  public ImportTableSchema(String tableFullName, JsonObject tableDefinition)
+      throws SchemaLoaderException {
+    String[] fullName = tableFullName.split("\\.", -1);
+    if (fullName.length < 2) {
```

Comment:
Is it expected if the length of `fullName` is 3 or more?

Reply:
Thanks. Should it be `length!= 2` right?

Reply:
Fixed in 0d3a2f8.